File: 0x08-python-more_classes/1-rectangle.py
#!/usr/bin/python3

"""" Define an class triangle """

class Rectangle:
    def __init__(self, width=0, height=0):
        self.__width = 0
        self.__height = 0


    def width(self):
        return self.__width

    @proprtysetter
    def width(self, value):
        self.__value = value
        if not isinstance(value, int):
            raise ValueError("width must be an integer")
        if width < 0:
            raise ValueError("width must be >= 0")

    def height(self):
        return self.__height

    def height(self, value):
        self.__value = value
        if not isinstance(value, int):
            raise ValueError("height must be an integer")
        if

File: 0x13-javascript_objects_scopes_closures/1-rectangle.js
#!/usr/bin/node
/**
 * An empty class Rectangle that define a rectangle
 * You must use the class notation for defining your class
 *i
 *run the main test
 *
 */

class Rectangle {
  /** export  class rectangle **/
  constructor (w, h) {
    /**
    *defining class constructor
    *width: Initialize the instance attribute width with the value of w
    *height: Initialize the instance attribute height with the value of h
    */
    this.width = w;
    this.height = h;
  }
}

module.exports = Rectangle;
